src/base/bittorrent/peeraddress.cpp

```diff
@@ -32,18 +32,18 @@
 
 using namespace BitTorrent;
 
-PeerAddress PeerAddress::parse(const QString &address)
+PeerAddress PeerAddress::parse(QStringView address)
```

Comment:
`const QStringView` (and other replacements of `const QString &`)
There exist `QStringView::chop()` which could modify the view.

Reply:
I hope I fixed it everywhere it isn't supposed to be modified.